api/v1/submissions/PKPSubmissionHandler.inc.php

```diff
@@ -745,6 +781,10 @@ public function editPublication($slimRequest, $response, $args)
         $primaryLocale = $publication->getData('locale');
         $allowedLocales = $submissionContext->getData('supportedSubmissionLocales');
 
+        if (!isset($primaryLocale)) {
+            $primaryLocale = $submissionContext->getData('primaryLocale');
+        }
```

Comment:
I don't think this should exist. If it does, that's a data error that we should correct elsewhere.

Reply:
Removed